### PR TITLE
add cluster based alternate schedules in prod

### PR DESCRIPTION
**Pull request by crimesp**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EM-3820

### Change description ###
Due to 2 clusters running at the same time, only 1 ingestor should be running - so use a cron schedule that alternates between the 2 clusters - during non business hours (9pm through 7am)

NOTE - this is the first time we have used cluster-based kustomization, please check approach is correct

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

**Review by yogesh-hullatti — APPROVED** [review]

File: k8s/prod/cluster-01-overlay/em/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: em
+patchesStrategicMerge:
+- ../../../namespaces/em/em-hrs-ingestor/prod-1.yaml
```
Comment (timja) [inline k8s/prod/cluster-01-overlay/em/kustomization.yaml:5]:
```suggestion
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization
namespace: em
bases:
- ../../../namespaces/em/em-icp/em-icp.yaml
patchesStrategicMerge:
- ../../../namespaces/em/em-icp/prod.yaml
- ../../../namespaces/em/em-hrs-ingestor/prod-1.yaml
```

**Review by timja — COMMENTED** [review]
don't you want this?

**Comment by crimesp** [comment]
hi @timja 

just checking this approach will work, as we have another overide for prod (setting env vars)
https://github.com/hmcts/cnp-flux-config/blob/74f94be20a849f2f3ae7ebcbb7a4fbb1f46dd902/k8s/namespaces/em/em-hrs-api/prod.yaml
and
https://github.com/hmcts/cnp-flux-config/blob/4d2a6637cda0a6ff4d6cf147fca874148c66b0b1/k8s/prod/common-overlay/em/kustomization.yaml#L12

will this work, or should we remove that kustomizer, and add the env vars into prod-0 and prod-1?

**Review by timja — APPROVED** [review]

**Comment by timja** [comment]
You can have multiple patches

You can run https://github.com/hmcts/cnp-flux-config/blob/master/k8s/prod/cluster-00-overlay/.flux.yaml#L4 locally to see what the generated yaml will look like

**Review by yogesh-hullatti — APPROVED** [review]

**Review by timja — APPROVED** [review]

**Pull request merged**